cfep-XX.md

```diff
@@ -0,0 +1,61 @@
+
+<table>
+<tr><td> Title </td><td> Guidelines for (Re)Naming Packages </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td>
+    Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;, Matthew R. Becker &lt;becker.mr@gmail.com&gt;, Axel Obermeier &lt;h.vetinari@gmx.com&gt;
+</td></tr>
+<tr><td> Created </td><td> Oct 26, 2025 </td></tr>
+<tr><td> Updated </td><td> Oct 26, 2025 </td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/admin-requests/pull/1711 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP describes the principles informing how and when to (re)name a package in conda-forge.
+Two cases are described depending on the status of the target name:
+
+- Unclaimed names: A new package or a package already present in conda-forge wants to use a new name currently unclaimed.
+- Names in use: A package, already present or about to be submitted, wishes to use a name already in use.
+
+## Specification
+
+This CFEP follows [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) in its use of certain keywords (e.g., MUST).
+
+### Unclaimed Names
+
+#### New Feedstocks & Name Squatting
+
+- Unclaimed package names are typically available on a first-come-first-served basis and can be claimed by submitting a recipe for a package via `staged-recipes` (or equivalent).
+- The core team reserves the right to prevent the use of certain names for any reason. Typical reasons the core team may prevent the use of a certain name include, but are not limited to, typo squatting, ambiguity, violations of the code of conduct, conflicts with important infrastructure, etc.
+- Pre-registering or squatting on a namespace in anticipation of future use is not allowed unless expressly approved by the core team via the consent of at least three core members, and no dissent (in case of dissent, this can be solved by a vote within the core team). The core team MUST be `@`-mentioned on the relevant PR/issue and this PR/issue MUST stay open for at least one week.
```

Comment:
```suggestion
- Pre-registering or squatting on a namespace in anticipation of future use is not allowed unless expressly approved by the core team via the consent of at least three core members, and no dissent (in case of dissent, this can be solved by a vote within the core team). The core team MUST be `@`-mentioned on the relevant PR/issue and this PR/issue MUST stay open for at least one week. For the duration of this process (from the time core gets pinged until an answer is given), no other packages can use the names in question.
```